climada/engine/impact_calc.py

```diff
@@ -101,13 +102,21 @@ def cover(self):
         if 'cover' in self.exposures.gdf.columns:
             return self.exposures.gdf['cover'].to_numpy()
 
-    def impact(self, save_mat=True):
+    def impact(self, save_mat=True, assign_centroids=True):
         """Compute the impact of a hazard on exposures.
 
         Parameters
         ----------
-        save_mat : bool
+        save_mat : bool, optional
             if true, save the total impact matrix (events x exposures)
+            Default: True
+        assign_centroids : bool, optional
+            indicates whether centroids are re-assigned to the self.exposures object
+            or kept from previous impact calculation with a hazard of the same hazard type.
+            Centroids assignment is an expensive operation; set this to ``False`` to save 
```

Comment:
I introduced trailing whitespace, which triggers new linter warnings. Sorry about that!
```suggestion
            Centroids assignment is an expensive operation; set this to ``False`` to save
```